Documentation/behind_firewall.rst

```diff
@@ -103,6 +107,7 @@ We describe how to use pip on a HPC cluster with restricts access to outside sou
         RemoteForward ddddd localhost:8899
 
 where ddddd is an arbitrary port number with 5 digits.
+ddddd is the port on the remote server that is forwarded to your local machine's port 8899.
```

Comment:
Maybe more a matter of taste, but I'd rather go with: "port 8899 of your local machine"